pkg/k8sutil/resource.go

```diff
@@ -168,6 +168,14 @@ func IsPodContainsTerminatedContainer(pod *corev1.Pod) bool {
 	return false
 }
 
+// IsPodContainsEvictedContainer returns true if pod status has an evicted reason false otherwise
+func IsPodContainsEvictedContainer(pod *corev1.Pod) bool {
```

Comment:
Since `pod.Status.Reason` is not an enum I's use `strings.Contains(pod.Status.Reason, "Evicted")`

Reply:
Good point:)